linuxNew/jdk/redhat/src/main/packaging/temurin/8/temurin-8-jdk.spec

```diff
@@ -0,0 +1,208 @@
+echo 'x /tmp/hsperfdata_*' > "%{buildroot}/usr/lib/tmpfiles.d/%{name}.conf"
+echo 'x /tmp/.java_pid*' >> "%{buildroot}/usr/lib/tmpfiles.d/%{name}.conf"
+
+%pretrans
+# noop
+
+%post
+if [ $1 -ge 1 ] ; then
+    update-alternatives --install %{_bindir}/java java %{prefix}/bin/java %{priority} \
+                        --slave %{_bindir}/jfr jfr %{prefix}/bin/jfr \
+                        --slave %{_bindir}/jjs jjs %{prefix}/bin/jjs \
+                        --slave %{_bindir}/jrunscript jrunscript %{prefix}/bin/jrunscript \
+                        --slave %{_bindir}/keytool keytool %{prefix}/bin/keytool \
+                        --slave %{_bindir}/pack200 pack200 %{prefix}/bin/pack200 \
+                        --slave %{_bindir}/rmid rmid %{prefix}/bin/rmid \
+                        --slave %{_bindir}/rmiregistry rmiregistry %{prefix}/bin/rmiregistry \
+                        --slave %{_bindir}/unpack200 unpack200 %{prefix}/bin/unpack200 \
+                        --slave %{_bindir}/jexec jexec %{prefix}/lib/jexec \
+                        --slave %{_bindir}/jspawnhelper jspawnhelper %{prefix}/lib/jspawnhelper \
+                        --slave  %{_mandir}/man1/java.1 java.1 %{prefix}/man/man1/java.1 \
+                        --slave  %{_mandir}/man1/jjs.1 jjs.1 %{prefix}/man/man1/jjs.1 \
+                        --slave  %{_mandir}/man1/jrunscript.1 jrunscript.1 %{prefix}/man/man1/jrunscript.1 \
+                        --slave  %{_mandir}/man1/keytool.1 keytool.1 %{prefix}/man/man1/keytool.1 \
+                        --slave  %{_mandir}/man1/pack200.1 pack200.1 %{prefix}/man/man1/pack200.1 \
+                        --slave  %{_mandir}/man1/rmid.1 rmid.1 %{prefix}/man/man1/rmid.1 \
+                        --slave  %{_mandir}/man1/rmiregistry.1 rmiregistry.1 %{prefix}/man/man1/rmiregistry.1 \
+                        --slave  %{_mandir}/man1/unpack200.1 unpack200.1 %{prefix}/man/man1/unpack200.1
+
+    update-alternatives --install %{_bindir}/javac javac %{prefix}/bin/javac %{priority} \
+                        --slave %{_bindir}/jaotc jaotc %{prefix}/bin/jaotc \
+                        --slave %{_bindir}/jar jar %{prefix}/bin/jar \
+                        --slave %{_bindir}/jarsigner jarsigner %{prefix}/bin/jarsigner \
+                        --slave %{_bindir}/javadoc javadoc %{prefix}/bin/javadoc \
+                        --slave %{_bindir}/javap javap %{prefix}/bin/javap \
+                        --slave %{_bindir}/jcmd jcmd %{prefix}/bin/jcmd \
+                        --slave %{_bindir}/jconsole jconsole %{prefix}/bin/jconsole \
+                        --slave %{_bindir}/jdb jdb %{prefix}/bin/jdb \
+                        --slave %{_bindir}/jdeprscan jdeprscan %{prefix}/bin/jdeprscan \
+                        --slave %{_bindir}/jdeps jdeps %{prefix}/bin/jdeps \
+                        --slave %{_bindir}/jhsdb jhsdb %{prefix}/bin/jhsdb \
+                        --slave %{_bindir}/jimage jimage %{prefix}/bin/jimage \
```

Comment:
Some of those binaries, like `jimage` and such are modularity only. So this needs revisiting.